services/ui/ws/server_window.cc

```diff
@@ -189,10 +189,10 @@ void ServerWindow::SetBoundsFromHostWindowManager(
   gfx::Rect old_bounds = bounds_;
   SetBoundsInternal(bounds, local_surface_id);
   if (bounds.origin() != old_bounds.origin()) {
-	// The root_'s origin mustn't be changed here, so that events are correctly
```

Comment:
in theory, this should go as a fixup of the patch that introduced it, so that we could squash them, and move commits around independently.